extension.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const vscode = require('vscode');
+const utilities = require('./lib/utilities.js');
+
+
+function activate(context) {
+    console.log('Congratulations, your extension "vsc-extension-ansible" is now active!');
+    var outputChannel = vscode.window.createOutputChannel("VSCode extension for Ansible");
+
+    var playbook = vscode.window.activeTextEditor.document.fileName;
+
+    let runpb = vscode.commands.registerCommand('extension.ansible-playbook', function () {
+        vscode.window.showInputBox({ value: playbook, prompt: 'Please input playbook name', placeHolder: 'playbook', password: false })
```

Comment:
we could just search for yml files in current dir (and subdirs optionally) and display list of choices?
we could also check if there are any open editors containing playbooks.

Reply:
default playbook is activetexteditor, if user not input playbook, will run that one. otherwise, just run one user input. when available yml file is long list, show list of choice is not so convinient, user need to scroll down a bit,